server/routerlicious/packages/services-core/src/tenant.ts

```diff
@@ -67,6 +67,17 @@ export enum KeyName {
 	key2 = "key2",
 }
 
+// This is tenantEncryptionKey version by year, it's append only.
+// We will add a new version each year.
+export enum EncryptionKeyVersion {
+	key2022 = "2022",
```

Comment:
To avoid making code change every year in OSS to introduce new version, can we fetch the two versions from config?

Reply:
It would be better to keep it as enum to regulate input. Probably what we can do is add some more key versions in advance. For example, we can add up to 2025 for now.